deps.ffmpeg/30-libvpx.ps1

```diff
@@ -1,12 +1,12 @@
 param(
     [string] $Name = 'libvpx',
     [string] $Version = '1.13.1',
-    [string] $Uri = 'https://github.com/webmproject/libvpx/archive/refs/tags/v1.13.1.zip',
-    [string] $Hash = "${PSScriptRoot}/checksums/v1.13.1.zip.sha256"
+    [string] $Uri = 'https://github.com/webmproject/libvpx.git',
+    [string] $Hash = '1672f4db71370e07e43e1e121cf2e92bf97ea4ff'
```

Comment:
v1.14.0-rc1 is out, so hopefully there's a stable tag that can be used soon.

Reply:
I am trying the PR, but i get many errors for example:
1 git clone :  https://github.com/PatTheMav/obs-deps/tree/windows-arm64-native, and checkout windows-arm64-native  branch,
2 try to run it locally:  .\Build-Dependencies.ps1 -Target 'arm64' -Configuration 'Debug' -PackageName 'ffmpeg' 

3 many libraries are built successfully, but some libraries fail, for example, mbedtls,

   1>Checking Build System
  Building Custom Rule C:/tools/alpaca/obs/PatTheMav-obs-deps/windows_build_temp/mbedtls/library/CMakeLists.txt
  asn1parse.c
  aesce.c
  aesni.c
  aria.c
C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.36.32532\include\immintrin.h(15,1): fatal error C1189: #error:  This header is spe
cific to X86, X64, ARM64, and ARM64EC targets [C:\tools\alpaca\obs\PatTheMav-obs-deps\windows_build_temp\mbedtls\build_arm64\library\mbedcrypto.vcxproj]
  base64.c


another questions, I have no ideay how to build these modules

•	libvpx: Uses shell support and make from MSYS2, compiled with MSBuild
•	x264: Uses shell support and make from MSYS2, compiled with cl
•	lame: Built natively using old (but still working) nmake script


@PatTheMav , could you guide me ?

thanks

Reply:
https://github.com/obsproject/obs-deps/pull/190#issuecomment-1886167857